Resolver.js

```diff
@@ -12,6 +12,16 @@ class Resolver {
         return this.resolve(name, target) || this.resolveInWrappedResolver(name, target);
     }
 
+    resolveRegiteredNames() {
```

Comment:
There is a typo in the Methode name `resolveRegiteredNames` should be `resolveRegisteredNames`